src/components/Checkout/OrderTotals.vue

```diff
@@ -1,85 +1,122 @@
 <template>
-	<div class="order-totals" data-testid="basket-order-totals-section">
+	<div class="tw-mb-1" data-testid="basket-order-totals-section">
 		<div
 			v-if="showPromoCreditTotal"
-			class="order-total tw-text-left md:tw-text-right"
+			class="tw-flex tw-flex-row tw-w-full tw-mb-2 tw-text-h3"
 			data-testid="basket-order-total"
 		>
-			<strong>Order Total: <span class="total-value">{{ itemTotal }}</span></strong>
```

Comment:
Can you please run kiva-tests against your local branch. I'm concerned that these class removals will affect those.

Reply:
I believe all the test currently use the test-id attribute, but I will verify 👍

Reply:
Thank you, I think and hope that's the case too!